### PR TITLE
Port majority of old-style connects

### PR DESCRIPTION
**Pull request by JBBgameich**
Done by clazy, so most connects involving overloaded signals are not yet ported.

### CODE REVIEW

**Comment by lnjX** [comment]
There's only something wrong with the transfer manager (and I think only with Qt <= 5.12).

**Comment by JBBgameich** [comment]
Noticed the coding style is somewhat broken now because of the automatic changed to the connects written in two lines. I'll fix that soon™️ (We really need clang-format support 😀).

File: src/server/QXmppServer.cpp [hunk 1]
```diff
@@ -553,9 +548,8 @@ bool QXmppServer::listenForClients(const QHostAddress &address, quint16 port)
     server->setLocalCertificate(d->localCertificate);
     server->setPrivateKey(d->privateKey);
 
-    check = connect(server, SIGNAL(newConnection(QSslSocket*)),
+    connect(server, SIGNAL(newConnection(QSslSocket*)),
                     this, SLOT(_q_clientConnection(QSslSocket*)));
-    Q_ASSERT(check);
```
Comment (lnjX) [inline src/server/QXmppServer.cpp]: Please don't remove the checks here, or port the connect.

File: src/server/QXmppServer.cpp [hunk 2]
```diff
@@ -617,9 +609,8 @@ bool QXmppServer::listenForServers(const QHostAddress &address, quint16 port)
     server->setLocalCertificate(d->localCertificate);
     server->setPrivateKey(d->privateKey);
 
-    check = connect(server, SIGNAL(newConnection(QSslSocket*)),
+    connect(server, SIGNAL(newConnection(QSslSocket*)),
                     this, SLOT(_q_serverConnection(QSslSocket*)));
-    Q_ASSERT(check);
```
Comment (lnjX) [inline src/server/QXmppServer.cpp]: Please don't remove the checks here, or port the connect.

File: examples/example_1_echoClient/example_1_echoClient.cpp [hunk 3]
```diff
@@ -32,10 +32,8 @@
 echoClient::echoClient(QObject *parent)
     : QXmppClient(parent)
 {
-    bool check = connect(this, SIGNAL(messageReceived(QXmppMessage)),
+    connect(this, SIGNAL(messageReceived(QXmppMessage)),
         SLOT(messageReceived(QXmppMessage)));
-    Q_ASSERT(check);
-    Q_UNUSED(check);
```
[the remark this answers is not in the document]
Comment (JBBgameich) [reply examples/example_1_echoClient/example_1_echoClient.cpp]: Oh sure, I wasn't aware there were still old ones left after you ported the overloaded ones. (And i did the latest changes automated again)

**Comment by lnjX** [comment]
ping

**Comment by lnjX** [comment]
Thank you very much! :)

**Pull request merged**